inc/Engine/Optimization/RUCSS/Warmup/ResourceFetcherProcess.php

```diff
@@ -64,6 +71,7 @@ protected function task( $resource ) {
 		}
 
 		if ( $this->resources_query->create_or_update( $resource ) ) {
+			$this->content_changed = true;
 			return ! $this->send_warmup_request( $resource );
```

Comment:
Will this ever return true? Presumably `send_warmup_request()` will (happy case) return true, but what if there is an API outage and it comes back false? Then `task()` returns `true`. Do we care? If it should really always return false, per the docblock, we should just `return false` here. Otherwise, maybe update the docblock to say what happens for true.